src/storage/store/column_chunk.cpp

```diff
@@ -122,6 +122,7 @@ ColumnChunk::ColumnChunk(
         break;
     }
     case PhysicalTypeID::VAR_LIST:
+    case PhysicalTypeID::INTERNAL_ID:
```

Comment:
Can we move the switch case into a separate function? This is becoming quite large now in the constructor.

Reply:
Sure. I'd already simplified it somewhat in #2102 ([here](https://github.com/kuzudb/kuzu/pull/2102/files#diff-b7ddec24858a04e0feec0e25f3a5dca30c0672beb96db4147b64e044a84dcad6R95-R144)), so I could move that change into this PR. Is that simplification enough, or would you prefer moving everything into a separate function?